### PR TITLE
Happy 10th birthday to /datum/recipe.

### PR DESCRIPTION
**Pull request by MistakeNot4892**
- Repathed recipes.
- Removed SScuisine.
- Cleaned up some recipe code.

### CODE REVIEW

File: code/datums/recipe.dm [hunk 1]
```diff
@@ -1,102 +1,87 @@
...
+
+/decl/recipe/proc/check_items(var/obj/container)
+	if(!length(items))
+		return TRUE
+	var/container_contents = container?.get_contained_external_atoms()
+	if(length(container_contents) < length(items))
+		return FALSE
+	var/list/needed_items = items.Copy()
+	for(var/itype in needed_items)
+		for(var/atom/movable/AM as anything in container_contents)
```
Comment (Lohikar) [inline code/datums/recipe.dm]: Why use a typed var at all if you're just doing an istype?

Comment (MistakeNot4892) [reply code/datums/recipe.dm]: Avoiding a cast.

Comment (MistakeNot4892) [reply code/datums/recipe.dm]: Nvm, just realized type is irrelevant there.

**Review by Lohikar — APPROVED** [review]
I haven't looked at the actual logic being used here, but the code looks good.

**Pull request merged**